soc/xtensa/intel_adsp/ace/power.c

```diff
@@ -163,9 +178,12 @@ __asm__(".align 4\n\t"
 	"  wsr   a1, WINDOWSTART\n\t"
 	"  wsr   a0, WINDOWBASE\n\t"
 	"  rsync\n\t"
-	"  movi  sp, d0i3_stack\n\t"
-	"  movi a2, 0x1000\n\t"
-	"  add sp, sp, a2\n\t"
+	"  movi  sp, z_interrupt_stacks\n\t"
```

Comment:
isn't `a1` the `sp` ? It is just confuse to me mixing nomenclature

Reply:
Yes, they are the same. But I don't see how it's confusing. In this part of code I'm calculating correct stack pointer.